### PR TITLE
ooniprobe 3.13.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
- license change relates to https://github.com/ooni/probe-cli/pull/446

**Review by iMichka — APPROVED** [review]

**Review by iMichka — DISMISSED** [review]
Need to check if the license is "`or-later`"

**Comment by iMichka** [comment]
```
    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.
```

**Comment by chenrui333** [comment]
That part is actually the same between [GPL-3.0-or-later ](https://spdx.org/licenses/GPL-3.0-or-later.html) and [GPL-3.0-only](https://spdx.org/licenses/GPL-3.0-only.html)

**Comment by chenrui333** [comment]
But I do assume this would be `-or-later`, just need some double confirmation from upstream on it first. :D

File: Formula/ooniprobe.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Ooniprobe < Formula
   desc "Network interference detection tool"
   homepage "https://ooni.org/"
-  url "https://github.com/ooni/probe-cli/archive/v3.10.1.tar.gz"
-  sha256 "2b81c14133f39ac91c4ea6761be7a27d768cd88989b52ae72376d1d7b69de322"
-  license "BSD-3-Clause"
+  url "https://github.com/ooni/probe-cli/archive/v3.13.0.tar.gz"
+  sha256 "a055aed8c2d0d898b7cdb843cf247cf3b593c8ac7045103c08b3088b7d4d1737"
+  license "GPL-3.0"
```
Comment (branchvincent) [inline Formula/ooniprobe.rb:6]: confirmed: https://github.com/ooni/probe-cli/pull/641

```suggestion
  license "GPL-3.0-or-later"
```

Comment (chenrui333) [reply Formula/ooniprobe.rb]: Nice!

**Review by branchv — APPROVED** [review]

**Pull request closed without merging**